egs/swbd/s5c/local/chain/tuning/run_blstm_6j.sh

```diff
@@ -185,6 +185,7 @@ if [ $stage -le 13 ]; then
     --trainer.optimization.initial-effective-lrate 0.001 \
     --trainer.optimization.final-effective-lrate 0.0001 \
     --trainer.optimization.momentum 0.0 \
+    --trainer.optimization.scale-learning-rate 0.1 \
```

Comment:
in general with these tuning scripts I prefer you to add a new script rather than edit the existing ones.. then you can document what was changed and what the results were.
But of course, if this idea doesn't work, it won't matter.